doc/source/whatsnew/v0.20.0.txt

```diff
@@ -49,6 +49,7 @@ Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
 - ``pd.read_excel`` now preserves sheet order when using ``sheetname=None`` (:issue:`9930`)
+- ``pandas.io.json.json_normalize`` If meta keys are not always present a new option to set errors="ignore" (:issue:`14583`)
```

Comment:
In ``pandas.io.json.json_normalize`` gained the option ``errors='ignore'|raise`` (in double-backticks); the default is ``raise`` and is backward compatible.